New package: ECCO v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: ECCO
- Repository: https://github.com/gaelforget/ECCO.jl
- Created by: @gaelforget
- Version: v0.0.1
- Commit: ea87296d36a53d03b57826e3c80bc939594a10c5
- Reviewed by: @gaelforget
- Reference: https://github.com/gaelforget/ECCO.jl/issues/1#issuecomment-2409075382
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835987ea25743eebebc736c747847a3d39fc5331e93829c6417ef156ca1b53811f60a145df7edb3d092017c196e148c884bead91edf0f517a1f5678d8ba4a819ca8b99864c81e7956027fdc3864fa9e0f2d4618ea104f04f4180c2b73c91ed1ca942c0ae198aeb24708697754ab4dcaa49221e3913bf35f04f56c97cf9f483419f5eb -->

gaelforget commented:
I would like my pull request to be manually merged instead of via AutoMerge. 

Explanation : the chosen name `ECCO.jl` stands for _Estimating the Circulation and Climate of the Ocean_ and is widely known in the climate and ocean science community by its acronym, `ECCO`.

- `ECCO` is a 40+ years running effort initiated by Pr. Carl Wunsch at MIT, and which now involves distributed academic projects accross the US ([MIT](https://github.com/MITgcm/MITgcm), [NASA JPL](https://www.ecco-group.org), [UCSD](https://www.ecco.ucsd.edu), UTA, WHOI), Europe (AWI, UH), and the UK (BAS). 
- extending the acronym would make it both (1) much harder to find by the targeted scientific community, and (2) a very long package name to type.
- if another package ended up called `ECCO.jl` later, this would be damageable to our climate science and community building effort (by making it even harder for users to find the present package, and possibly turn them off from bridging to Julia, which is one of the goal of this package).

JuliaRegistrator committed:
New package: ECCO v0.0.1

UUID: 485acfd6-c8ad-4ff9-b233-a5c7cab2dfb0
Repo: https://github.com/gaelforget/ECCO.jl.git
Tree: 066d9899772dbe3e3a1d36d78d8caf22321a2d03

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

goerz commented:
I like the name, and I’d definitely support merging this. In my opinion, pronounceable acronyms should always be okay (it’s just hard for the bot to distinguish pronounceable from unpronounceable acronyms)

It’s not up to me, though. The 5 letter minimum rule can only be overridden with a manual merge from a registry maintainer.

You have to fix the compat issues first, though. Those are mandatory.

gaelforget commented:
> I like the name, and I’d definitely support merging this. In my opinion, pronounceable acronyms should always be okay (it’s just hard for the bot to distinguish pronounceable from unpronounceable acronyms)
> 
> It’s not up to me, though. The 5 letter minimum rule can only be overridden with a manual merge from a registry maintainer.

Thanks for the feedback and supporting words! Makes me hopeful that a registry maintainer will agree with you and merge

> You have to fix the compat issues first, though. Those are mandatory.

should be all set with compat now I think, following

- https://github.com/gaelforget/ECCO.jl/commit/ea87296d36a53d03b57826e3c80bc939594a10c5
- https://github.com/gaelforget/ECCO.jl/commit/3d8305377a6c1535eaca6c99cd6d3065c5694e72

Or am I overlooking something?

gaelforget commented:
Hi @goerz & @vchuravy 

just giving this a ping to bring it back up on your radar. Would be awesome if this could be merged today. 

I am almost ready to release the next version, v0.1.1, which I am hoping to do live as a demo tomorrow (at this [hackweek](https://ecco-hackweek.github.io/ecco-2024/)).

goerz commented:
I don't have merge permission, unfortunately, but yes, this _should_ be merged as soon as possible, since the three-day period is up. If @vchuravy can't help you out with this, maybe ping the `#pgk-registration` channel on Slack.

> I am almost ready to release the next version, v0.1.1

That seems like a version jump… Do you mean `v0.1.0`? In any case, do not try to register a new version before this PR is merged.

gaelforget commented:
> I don't have merge permission, unfortunately, but yes, this _should_ be merged as soon as possible, since the three-day period is up. If @vchuravy can't help you out with this, maybe ping the `#pgk-registration` channel on Slack.

Cool. Will give it a minute for now

> > I am almost ready to release the next version, v0.1.1
> 
> That seems like a version jump… Do you mean `v0.1.0`? In any case, do not try to register a new version before this PR is merged.

Good point. Did mean `v0.1.0`
